# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../procps/termios.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

At leat with dietlibc termios.h is not included thru the other includes of
procps, thus the storage size of the winsize struct is not known in several
files. We just include it globally to make sure it is available in any file.

  - Rene Rebe <rene@exactcode.de>

--- ./proc/procps.h.vanilla	2005-07-28 17:48:01.000000000 +0200
+++ ./proc/procps.h	2005-07-28 17:48:08.000000000 +0200
@@ -109,4 +109,6 @@
 #define HIDDEN_ALIAS(x) extern __typeof(x) x##_direct __attribute__((alias(#x)))
 #endif
 
+#include <termios.h>
+
 #endif
